### PR TITLE
Feat/game page redesign

### PR DESCRIPTION
**Pull request by u22520016**
I added the xbox/playstation icons and systems requirements (best way to see all system requirements for a game is to check GTA for a full list and The Witcher 3 for no list)

### CODE REVIEW

File: lib/view/components/feed/event_invite.dart [hunk 1]
```diff
@@ -67,12 +67,14 @@ class _EventInvitationState extends State<EventInvitation> {
                         children: [
                           Text('${widget.inviterName} invited you to join',
                               style:
-                                  const TextStyle(fontWeight: FontWeight.bold)),
+                                  const TextStyle(fontWeight: FontWeight.bold),
+                              overflow: TextOverflow.ellipsis,),
```
Comment (MonikaTheiss) [inline lib/view/components/feed/event_invite.dart:71]: I see you tried to handle the overflow error on the event invites, however when I run it on my side I still get an overflow. Will send you a screen shot

**Review by MonikaTheiss — COMMENTED** [review]
Looks good! Thank you 😄 I left a comment on the overflow of the event invite, and sent you a screenshot so you can see whats going on. Once that has been addressed I can approve the pull request.

**Review by pieryvdl — COMMENTED** [review]
Hi Franco! Thanks for changing the icons, and adding the system requirements! It looks great. I agree with @MonikaTheiss , I will send a screenshot here. 
image
The button for the start playing is overflowing. But for the bottom overflow, you can look at the messaging_page.dart on the component that I build. I used this package "auto_size_text: ^3.0.0" to autosize the text. Hopefully that will help solve the problem. The rest of the work is great! Thank you for your hard work.

**Review by MonikaTheiss — APPROVED** [review]
@u22520016 informed me that he will address the overflow issues on the feed page in a different pull request , he is currently working on it, so I will approve this as this pull request is for the specific game page. 
Thank you @u22520016 for adding the system requirements!

**Review by pieryvdl — APPROVED** [review]
In that case I will approve the changes. Thanks for letting me know @MonikaTheiss. 
Thank you @u22520016 it looks great!

**Pull request merged**